pages/web_cloud/web_hosting/cms-update-password-admin/guide.fr-fr.md

```diff
@@ -0,0 +1,123 @@
+---
+title: "Comment changer le mot de passe administrateur de mon CMS (WordPress, PrestaShop, Joomla!, Drupal)"
+excerpt: "Découvrez comment modifier le mot de passe administrateur de votre CMS WordPress, PrestaShop, Joomla! et Drupal"
```

Comment:
```suggestion
excerpt: "Découvrez comment modifier le mot de passe administrateur de votre CMS directement via l’interface d’administration du CMS ou en utilisant phpMyAdmin depuis l’espace client OVHcloud"
```
To avoid repetition of the title